manageability-and-operations/observability-and-manageability/external-database-enablement/root_module/variables.tf

```diff
@@ -0,0 +1,5 @@
+variable "compartment_ocid" {
+  description = "Compartment OCID"
+  type        = string
+  default     = "ocid1.compartment.oc1..XXXXXX"
```

Comment:
Remove line.

Reply:
Removed placeholder OCIDs in the three variable.tf-files specified. See commit [c8b0810](https://github.com/oracle-devrel/technology-engineering/pull/856/commits/c8b0810732d9704a0e2662aaa9bbcc2b859acffb)